src/basic/init/e394.basic_cold_start.s

```diff
@@ -8,23 +8,6 @@
 
 basic_cold_start:
 
-!ifdef ROM_LAYOUT_STD { ; skip on non-sgtandard ROM layouts - they produce single image nevertheless
```

Comment:
Never ever move this code out of the $E000-$FFFF block, or it won't work correctly!

The idea is that we should display a panic screen if the user tries to use BASIC and KERNAL images which do not match; for this purpose we can't rely on `jmp basic_cold_start_internal` call, as the target address will be different for different builds.

Reply:
Understood. I would need advice on what to do in this case since the routine is to big to fit in the gap now.

Reply:
You'll need to have a separate variant of this code for the 38K memory model, where the check is a separate routine. See files in `src/basic/memory` and `src/basic/engine_variables` for some examples. The `;; #LAYOUT# STD *       #TAKE-HIGH` directive at the top of the file enforces the BASIC code from the file to go to the $E000-$FFFF block (more precisely, to the BASIC part of the block).

You can read about the code segments in the https://github.com/MEGA65/open-roms/blob/master/doc/Development.md file.